src/Pages/History.js

```diff
@@ -0,0 +1,55 @@
+import { useLibraryContext } from "../Context/LibraryContext";
```

Comment:
In History.js, Home.js, Playlist.js and other such files import things from index.js directly instead of this "../Components/MediaPlayer/VideoCardHorizontal"

Reply:
I have not used index.js as, as per my understanding it is used for exporting pages rather than components. Please do correct me, if I am wrong.

Reply:
As per my understanding Pages comprises of Components, which are imported to make app functional. 
![components](https://user-images.githubusercontent.com/66156650/117188964-3e62f000-adfb-11eb-8433-863852db6d49.png)
 Refer Image for what I want to say rather than getting other misunderstandings. This makes code look clean